Library/Homebrew/formula_creator.rb

```diff
@@ -5,49 +5,58 @@
 require "erb"
 
 module Homebrew
+  module_function
+
+  def url2name(url)
+    odebug "url: #{url}"
+    stem = Pathname.new(url).stem
+    odebug "stem: #{stem}"
+    # special cases first
+    name = if stem.start_with? "index.cgi"
+      # http://www.codesrc.com/gitweb/index.cgi?p=libzipper.git;a=summary
+      # https://github.com/Homebrew/brew/pull/9459
```

Comment:
I don't understand this URL?

Reply:
Support for parsing `http://www.codesrc.com/gitweb/index.cgi?p=libzipper.git;a=summary` in the wrong place was added in https://github.com/Homebrew/brew/pull/9459

The comment could be improved, I agree. Or moved into commit message. I will do it on next rebase.